web/modules/custom/dbcdk_community/src/Plugin/Block/ProfileQuarantinesBlock.php

```diff
@@ -108,14 +108,7 @@ public function build() {
       ];
       $profile = $this->profileApi->profileFindOne(json_encode($profile_filter));
 
-      // Fetch quarantines.
-      $quarantine_filter = [
-        'order' => 'end DESC',
-        'where' => [
-          'quarantinedId' => $profile->getId(),
-        ],
-      ];
-      $quarantines = (array) $this->quarantineApi->quarantineFind(json_encode($quarantine_filter));
+      $quarantines = (array) $this->profileApi->profilePrototypeGetQuarantines($profile->getId(), json_encode(['order' => 'end DESC']));
```

Comment:
I might not completely agree with using Prototype methods where we don't necessarily have to since they are declared as a "state of not done" (or that is my understanding of it) - but since we're manually updating the Community Service Client, then this will not cause any sudden problems so I don't mind it